### ISSUE TITLE
undefined symbol: av_top_index

### ISSUE THREAD

**Issue by eserte** [merkys/Graph-Nauty#1]
It seems that the module is using a perl function not defined for perl 5.16 and older:
```
#   Failed test 'use Graph::Nauty;'
#   at t/01_use.t line 5.
#     Tried to use 'Graph::Nauty'.
#     Error:  Can't load '/home/cpansand/.cpan/build/2020050108/Graph-Nauty-0.1.0-3uvHGp/blib/arch/auto/Graph/Nauty/Nauty.so' for module Graph::Nauty: /home/cpansand/.cpan/build/2020050108/Graph-Nauty-0.1.0-3uvHGp/blib/arch/auto/Graph/Nauty/Nauty.so: undefined symbol: av_top_index at /opt/perl-5.14.4/lib/5.14.4/x86_64-linux/DynaLoader.pm line 190.
#  at t/01_use.t line 5.
# Compilation failed in require at t/01_use.t line 5.
# BEGIN failed--compilation aborted at t/01_use.t line 5.
# Looks like you failed 1 test of 1.
t/01_use.t ....... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```
A possible fix is to use ppport.h.

**Comment by merkys** [comment]
Thanks for letting me know. I may as well look for a more stable way to get the number of elements in Perl array.

**Comment by eserte** [comment]
I think adding ppport.h is the easiest solution.

**Comment by merkys** [comment]
Fixed in 4853193f754e1cde8e65ddf62ac849fcd07134f4.

**Comment by eserte** [comment]
It seems that an `#include "pport.h"` is still missing.

**Comment by eserte** [comment]
@merkys: the #include is still missing.

**Comment by merkys** [comment]
OK, I will look into it.

**Comment by merkys** [comment]
Fixed in 48e06f7eda756b4732b81233f251aefa0a825e9e.
